cli/compose/loader/loader.go

```diff
@@ -45,27 +45,36 @@ func Load(configDetails types.ConfigDetails) (*types.Config, error) {
 	if len(configDetails.ConfigFiles) < 1 {
 		return nil, errors.Errorf("No files specified")
 	}
-	if len(configDetails.ConfigFiles) > 1 {
-		return nil, errors.Errorf("Multiple files are not yet supported")
-	}
 
-	configDict := getConfigDict(configDetails)
-	configDetails.Version = schema.Version(configDict)
+	configs := []*types.Config{}
 
-	if err := validateForbidden(configDict); err != nil {
-		return nil, err
-	}
+	for _, file := range configDetails.ConfigFiles {
+		configDict := file.Config
+		configDetails.Version = schema.Version(configDict)
```

Comment:
I think this should only set the version from the first config, and error if any other config does not match that version.